src/Castle.Windsor.Extensions.DependencyInjection/Castle.Windsor.Extensions.DependencyInjection.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

Comment:
Does `Castle.Windsor.Extensions.DependencyInjection` run on .NET Framework? I thought the MS libraries were .NET Core only.

Reply:
That’s odd, must have been on autopilot, don’t _think_ this was intentional. Will check later.

Reply:
Wasn't intentional; reverting.

Reply:
Resolved - rebased and confused myself, but have got rid of the addition.

Reply:
The unit test project is still changed from `net5.0` to `netcoreapp3.1`.

Reply:
That's intentional, for two reasons:
1) `net5.0` is out of support
2) `netcoreapp3.1` is preferable to `net6.0`, as otherwise adding `net6.0` to the base project would mean the `netstandard2.0` output is no longer tested.

Reply:
Thanks for the explanation 👍